From 52536539ed1815c2a5c8e86e96b095c5bc332102 Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniel@fooishbar.org>
Date: Tue, 29 Jul 2008 17:52:11 +0300
Subject: [PATCH 19/41] EXA: Remove unnecessary #includes

There wasn't actually any font code here, so no problem.
(cherry picked from commit e882ee7056f370e0619d137b4ec3973ecb4e3479)
---
 exa/exa.c           |    2 --
 exa/exa_migration.c |    2 --
 2 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/exa/exa.c b/exa/exa.c
index 5607baa..5ab933f 100644
--- a/exa/exa.c
+++ b/exa/exa.c
@@ -35,8 +35,6 @@
 #include <stdlib.h>
 
 #include "exa_priv.h"
-#include <X11/fonts/fontstruct.h>
-#include "dixfontstr.h"
 #include "exa.h"
 #include "cw.h"
 
diff --git a/exa/exa_migration.c b/exa/exa_migration.c
index 9dd19c6..cf22d6f 100644
--- a/exa/exa_migration.c
+++ b/exa/exa_migration.c
@@ -33,8 +33,6 @@
 #include <string.h>
 
 #include "exa_priv.h"
-#include <X11/fonts/fontstruct.h>
-#include "dixfontstr.h"
 #include "exa.h"
 #include "cw.h"
 
-- 
1.6.0.4

